### PR TITLE
hep: infer curated flag from 500__a

### PR DESCRIPTION
**Pull request by jacquerie**
Closes https://github.com/inspirehep/inspire-dojson/issues/87.

### CODE REVIEW

File: inspire_dojson/hep/rules/bd5xx.py [hunk 1]
```diff
@@ -42,9 +42,20 @@
 def public_notes(self, key, value):
     """Populate the ``public_notes`` key.
 
-    Also populates the ``thesis_info`` key through side effects.
+    Also populates the ``curated`` and ``thesis_info`` keys through side effects.
     """
+    def _means_not_curated(public_note):
+        return any(public_note == el for el in [
```
Comment (michamos) [inline inspire_dojson/hep/rules/bd5xx.py]: `return public_note in [...]`?

File: inspire_dojson/hep/rules/bd5xx.py [hunk 2]
```diff
@@ -53,16 +64,29 @@ def public_notes(self, key, value):
...
     self['thesis_info'] = thesis_info
     return public_notes
 
 
+@hep2marc.over('500', '^curated$')
+def curated2marc(self, key, value):
+    result = self.get('500', [])
+
+    if value is False:
+        result.insert(0, {'a': '* Brief entry *'})
```
Comment (michamos) [inline inspire_dojson/hep/rules/bd5xx.py]: this is wrong: core -> Temporary, non-core -> Brief.

File: inspire_dojson/hep/rules/bd5xx.py (hunk 1 above)
Comment (jacquerie) [reply inspire_dojson/hep/rules/bd5xx.py]: Whoops, right!

File: inspire_dojson/hep/rules/bd5xx.py (hunk 2 above)
Comment (jacquerie) [reply inspire_dojson/hep/rules/bd5xx.py]: Oh, so "Temporary" and "Brief" mean different things, and I need to know if the record is also CORE to know which to choose?

Comment (michamos) [reply inspire_dojson/hep/rules/bd5xx.py]: yes, core records are curated eventually and lose their temporary flag. non-core records on the other hand are generally not curated, so their state is not temporary.

**Pull request merged**